Formula/minetest.rb

```diff
@@ -105,8 +115,8 @@ def caveats
       "~/Library/Application Support/minetest/", respectively (you may have
       to create those folders first).
 
-      If you would like to start the Minetest server from a terminal, run
-      "#{prefix}/minetest.app/Contents/MacOS/minetest --server".
+      To start minetest, from a terminal run
+      "#{prefix}/minetest.app/Contents/MacOS/minetest".
```

Comment:
Are the old instructions wrong?

Reply:
`--server` does start the headless dedicated server, but this is unlikely to be the action that users will want after installing. Running `--server` without specifying a game and configuration will just abort with an error. Most first-time users will want to start the graphical interface, which requires omitting `--server`.

Users who finish installation are often confused what to do next, and show up in minetest's IRC/discord to ask for help. They often look for an icon in the dock or Applications folder, and can't find anything. If you could point me towards a way to make such an icon appear automatically, that would be even better!